tests/e2e/.ipfs/version

```diff
@@ -1 +1 @@
-12
+15
```

Comment:
What version of Kubo are we upgrading to? We need to replace the `dev-dependencies/ipfs` binary in `dev-forta-releases` bucket with that, to keep the E2E testing working in the CI.